practitioner-service/pom.xml

```
@@ -3,21 +3,22 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">  
   <modelVersion>4.0.0</modelVersion>  
   <parent> 
-    <groupId>org.springframework.boot</groupId>  
-    <artifactId>spring-boot-starter-parent</artifactId>  
-    <version>2.3.6.RELEASE</version>  
-    <relativePath/>  
+    <groupId>uk.ac.ox.ctsu</groupId>  
+    <artifactId>java-base-app</artifactId>  
+    <version>0.0.1-SNAPSHOT</version>  
+    <relativePath>../</relativePath>  
     <!-- lookup parent from repository --> 
   </parent>  
   <groupId>uk.ac.ox.ndph.arts</groupId>
   <artifactId>practitioner_service</artifactId>  
   <version>0.0.1-SNAPSHOT</version>  
-  <name>practitioner-service</name>  
+  <name>practitioner_service</name>  
```

This is a friendly name, so code be:
```suggestion
  <name>Practitioner Service</name>  
```